Extensions/TextObject/Extension.cpp

```diff
@@ -64,6 +64,7 @@ void DeclareTextObjectExtension(gd::PlatformExtension& extension) {
           "string",
           gd::ParameterOptions::MakeNewOptions().SetDescription(
               _("Text to compare to")))
+      .AddParameter("yesorno", _("Case insensitive"))
```

Comment:
This parameter could be added by `UseStandardRelationalOperatorParameters` to add the parameter on any text conditions.

Reply:
That was right! I did it and commited now.